spec/requests/user_spec.rb

```diff
@@ -0,0 +1,7 @@
+require 'rails_helper'
+
+RSpec.describe 'Users', type: :request do
+  describe 'GET /index' do
+    pending "add some examples (or delete) #{__FILE__}"
+  end
+end
```

Comment:
- [ ]  Please kindly include tests for this request

Reply:
@AnselemOdims  Thank you very much for your review on our PR and good points to improve our RecipeApp, we have implemented the UI changes required and additional tests.

![3](https://user-images.githubusercontent.com/40719899/208240402-e3a329b6-45df-4ae6-a311-9601173bbb82.gif)


The tests for model specs and request specs

![image](https://user-images.githubusercontent.com/40719899/208240492-0ea3d7a2-1cc4-4ec0-8e2c-01aa12e77817.png)